lib/shark_client.js

```diff
@@ -129,6 +127,21 @@ function _request(opts, cb) {
     }
 }
 
+/*
+ * Use information about the object to craft the storage path.
+ */
+function storagePath(opts) {
+    var storageLayoutVersion = '/v' + opts.storageLayoutVersion + '/';
+    var owner = opts.creator || opts.owner;
```

Comment:
Showing my ignorance: is there really an "creator *or* owner" thing here in the buckets code path?

Reply:
No, this is a vestige of the muskie codebase. I'll take that out.